CHANGES.md

```diff
@@ -121,6 +121,8 @@ To be released.
  -  Fixed a bug that `IActionContext.Random` had been possible to generated
     equivalent results between actions of different transactions in
     a `Block<T>`.  [[#519]]
+ -  Fixed a bug where a forked chain would not be deleted when an exception
+    occurred during fetching block from other peers.  [[#527], [#535]]
```

Comment:
Is there no pull request reference? and why is there #535 reference?

Reply:
My mistake. :cry: I'll fix it in another PR.